src/include/storage/store/table.h

```diff
@@ -141,6 +141,7 @@ class Table {
 
     common::TableType getTableType() const { return tableType; }
     common::table_id_t getTableID() const { return tableID; }
+    std::string getTableName() const { return tableName; }
```

Comment:
Is this used anywhere?

Reply:
yup, to get tableName for localTable during exception message